packages/devtools_app/test/primitives/utils_test.dart

```diff
@@ -666,15 +667,17 @@ void main() {
           'key2': '123',
         }),
       );
-      expect(
-        devToolsQueryParams(
-          'http://localhost:9101/#/app-size?key=value.json&key2=123',
-        ),
-        equals({
-          'key': 'value.json',
-          'key2': '123',
-        }),
-      );
+      for (final meta in ScreenMetaData.values) {
+        expect(
+          devToolsQueryParams(
+            'http://localhost:9101/#/${meta.id}?key=value.json&key2=123',
```

Comment:
nit: you can remove the `/#` fragment too now. That is not part of the DevTools URL anymore.

Reply:
👍